### PR TITLE
Set Presto User in header of HTTP request of Presto Node Info API

### PR DESCRIPTION
**Pull request by harmandeeps**

### CODE REVIEW

File: rubix-presto/src/main/java/com/qubole/rubix/presto/PrestoClusterManager.java [hunk 1]
```diff
@@ -38,11 +38,14 @@
...
 import java.util.Set;
 
+import static java.util.Objects.requireNonNull;
+
 /**
  * Created by stagra on 14/1/16.
  */
 public class PrestoClusterManager extends ClusterManager
 {
+  private static final String DEFAULT_USER = "qubole";
```
Comment (stagraqubole) [inline rubix-presto/src/main/java/com/qubole/rubix/presto/PrestoClusterManager.java:48]: change user-name to rubix

File: rubix-prestosql/src/main/java/com/qubole/rubix/prestosql/PrestoClusterManager.java [hunk 2]
```diff
@@ -50,6 +50,7 @@
  */
 public class PrestoClusterManager extends ClusterManager
 {
+  private static final String DEFAULT_USER = "qubole";
```
Comment (stagraqubole) [inline rubix-prestosql/src/main/java/com/qubole/rubix/prestosql/PrestoClusterManager.java:53]: same here

**Review by stagraqubole — CHANGES_REQUESTED** [review]

File: rubix-presto/src/main/java/com/qubole/rubix/presto/PrestoClusterManager.java (hunk 1 above)
Comment (harmandeeps) [reply rubix-presto/src/main/java/com/qubole/rubix/presto/PrestoClusterManager.java]: done

File: rubix-prestosql/src/main/java/com/qubole/rubix/prestosql/PrestoClusterManager.java (hunk 2 above)
Comment (harmandeeps) [reply rubix-prestosql/src/main/java/com/qubole/rubix/prestosql/PrestoClusterManager.java]: done

**Review by stagraqubole — APPROVED** [review]

**Pull request merged**